### PR TITLE
[SIG-3484] Remove icon url when posting incident

### PR DESCRIPTION
**Pull request by aditudorache**
This PR restructures the item structure by removing the iconUrl property as this is not expected by the api.

### CODE REVIEW

File: src/signals/incident/components/form/ContainerSelect/components/Selector/index.tsx [hunk 1]
```diff
@@ -215,7 +214,7 @@ const Selector = () => {
                 onClose={handleLegendCloseButton}
                 variant={panelVariant}
                 title="Legenda"
-                items={meta.featureTypes.map(featureType => ({
+                items={meta.featureTypes.filter(({ label }) => label !== 'Onbekend').map(featureType => ({
```
Comment (MauritsR) [inline src/signals/incident/components/form/ContainerSelect/components/Selector/index.tsx:217]: You can use the `unknownFeatureType` defined in the beginning of the file to get this label

File: src/signals/incident/definitions/wizard-step-5-samenvatting.js [hunk 2]
```diff
@@ -44,7 +45,8 @@ export const renderPreview = ({ render: renderFunc, meta }) => {
       return Label;
 
     case 'ContainerSelectRenderer':
-      return PreviewComponents.ContainerListPreview;
+      return Preview;
+      // return props => PreviewComponents.ContainerListPreview({ ...props, featureTypes: meta.featureTypes });
```
Comment (MauritsR) [inline src/signals/incident/definitions/wizard-step-5-samenvatting.js:49]: Forgot to remove?

**Review by MauritsR — CHANGES_REQUESTED** [review]

File: src/signals/incident/definitions/wizard-step-5-samenvatting.js (hunk 2 above)
Comment (aditudorache) [reply src/signals/incident/definitions/wizard-step-5-samenvatting.js:49]: Thanks!

File: src/signals/incident/components/form/ContainerSelect/components/Selector/index.tsx (hunk 1 above)
Comment (aditudorache) [reply src/signals/incident/components/form/ContainerSelect/components/Selector/index.tsx:217]: The `unknownFeatureType` is moved to the metadata so I removed from here!

**Review by MauritsR — APPROVED** [review]

**Pull request merged**